### PR TITLE
Wait for commands before calling them

### PR DESCRIPTION
**Pull request by rtimush**
Commands may appear on UI a bit later then instance becomes active, so we first wait for a command and only then call it.

### CODE REVIEW

**Comment by rtimush** [comment]
@vasichkin please review

File: qubell/api/private/instance.py [hunk 1]
```diff
@@ -281,8 +281,23 @@ def destroyed(self, timeout=3):  # Shortcut for convenience. Timeout = 3 min (as
...
             parameters = {}
-        log.info("Running workflow %s on instance id=%s" % (name, self.id))
+        log.info("Running command %s on instance id=%s component=%s" % (name, self.id, component_path))
         log.debug("Parameters: %s" % parameters)
+        if name != "destroy" or component_path is not None:
+            if component_path is None:
+                path = []
+            else:
+                path = component_path.split(".")
+            @retry(tries=5, delay=1, backoff=1.5, retry_exception=(AssertionError, StopIteration, KeyError))
```
Comment (dmakhno) [inline qubell/api/private/instance.py]: StopIteration? have never met this exception, where does it appear?

Comment (rtimush) [inline qubell/api/private/instance.py]: It is thown from next(<empty generator>)

File: qubell/api/private/instance.py [hunk 2]
```diff
@@ -281,8 +281,23 @@ def destroyed(self, timeout=3):  # Shortcut for convenience. Timeout = 3 min (as
...
+            def eventually_has_command():
+                log.debug("Verifying that command %s exists" % name)
+                component = self.json()
+                for x in path:
+                    component = next(c for c in component['submodules'] if c.get('componentId', c.get('name')) == x)
+                workflows = [wf['name'] for wf in component.get('workflowsInfo', {}).get('availableWorkflows', {})]
+                commands = [wf['name'] for wf in component.get('commands', {})]
+                assert (name in workflows) or (name in commands)
+            eventually_has_command()
```
Comment (dmakhno) [inline qubell/api/private/instance.py]: do we need mocked test for this logic?

**Comment by vasichkin** [comment]
@rtimush, is it hard to support this in lime tests? How much tests affected?
I think we will loose feeling of sluggishness of the platform once 'eventually's  like this merged into client.
@dmakhno, wdyt?

**Comment by rtimush** [comment]
@vasichkin Now we have just a couple of tests affected, but any future test that calls commands on components will be affected too.

**Comment by vasichkin** [comment]
@rtimush Yes, i do understand this. Also, i see platform responding slower and slower, so bugs like this became visible. If code moved to client, slowdown became 'invisible' for lime tests. That is what i'm afraid of. We are not monitoring platform response time yet.
Okay. 
I will refactor this later. Could be merged.

**Comment by vasichkin** [comment]
Please, commit changes to develop branch

**Pull request closed without merging**